java-manta-client-unshaded/src/main/java/com/joyent/manta/client/multipart/JobsMultipartManager.java

```diff
@@ -303,12 +294,12 @@ MantaMultipartUploadPart uploadPart(final JobsMultipartUpload upload,
         Validate.notNull(entity, "Upload entity must not be null");
 
         final String path = multipartPath(upload.getId(), partNumber);
-        final HttpPut put = connectionFactory.put(path);
+        final HttpPut put = httpHelper.getRequestFactory().put(path);
         put.setEntity(entity);
 
         final int expectedStatusCode = HttpStatus.SC_NO_CONTENT;
 
-        try (CloseableHttpResponse response = connectionContext.getHttpClient().execute(put, context)) {
+        try (CloseableHttpResponse response = httpHelper.getConnectionContext().getHttpClient().execute(put, context)) {
```

Comment:
This is a long chain of linked objects. Maybe break out the context or client as its own variable?